dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,56 @@
+Feature: AI Lab
+
+  @as_student
+  Scenario: Training a model with an in-house dataset
+    Given I am on "http://studio.code.org/s/allthethings/lessons/43/levels/1"
+    And I wait to see the dynamic instructions contain "selectDataset" key
+    And I close the instructions overlay if it exists
```

Comment:
Lower priority, but since we know the overlay should be showing, I wonder if we could do a version of this function without the `"if it exists"` to ensure that the overlay really is showing.